api/src/test/java/org/pmiops/workbench/api/DiskControllerTest.java

```diff
@@ -0,0 +1,331 @@
+  // the same.
+  private static final String WORKSPACE_ID = "myfirstworkspace";
+  private static final String WORKSPACE_NAME = "My First Workspace";
+  private static final String LOGGED_IN_USER_EMAIL = "bob@gmail.com";
+  private static final String OTHER_USER_EMAIL = "alice@gmail.com";
+  private static final String BUCKET_NAME = "workspace-bucket";
+  private static final Instant NOW = Instant.now();
+  private static final FakeClock CLOCK = new FakeClock(NOW, ZoneId.systemDefault());
+  private static final String API_HOST = "api.stable.fake-research-aou.org";
+  private static final String API_BASE_URL = "https://" + API_HOST;
+  private static final String BIGQUERY_DATASET = "dataset-name";
+
+  private static WorkbenchConfig config = new WorkbenchConfig();
+  private static DbUser user = new DbUser();
+
+  @TestConfiguration
+  @Import({
+    DiskController.class,
+    CohortMapperImpl.class,
+    CohortReviewMapperImpl.class,
+    ConceptSetMapperImpl.class,
+    DataSetMapperImpl.class,
+    FirecloudMapperImpl.class,
+    WorkspaceMapperImpl.class,
+    CommonMappers.class,
+    PublicInstitutionDetailsMapperImpl.class,
+    UserServiceTestConfiguration.class,
+    LeonardoMapperImpl.class,
+    LeonardoNotebooksClientImpl.class,
+    NotebooksRetryHandler.class,
+    LeonardoRetryHandler.class,
+    NoBackOffPolicy.class,
+    AccessTierServiceImpl.class,
+  })
+  @MockBean({
+    AccessModuleService.class,
+    ConceptSetService.class,
+    CohortService.class,
+    MailService.class,
+  })
+  static class Configuration {
+
+    @Bean
+    @Scope("prototype")
+    public WorkbenchConfig workbenchConfig() {
+      return config;
+    }
+
+    @Bean
+    Clock clock() {
+      return CLOCK;
+    }
+
+    @Bean
+    @Scope("prototype")
+    DbUser user() {
+      return user;
+    }
+
+    @Bean
+    Random random() {
+      return new FakeLongRandom(123);
+    }
+  }
+
+  @Captor private ArgumentCaptor<LeonardoUpdateDiskRequest> updateDiskRequestCaptor;
+
+  @MockBean AdminActionHistoryDao mockAdminActionHistoryDao;
+  @MockBean LeonardoRuntimeAuditor mockLeonardoRuntimeAuditor;
+  @MockBean ComplianceService mockComplianceService;
+  @MockBean DirectoryService mockDirectoryService;
+  @MockBean FireCloudService mockFireCloudService;
+  @MockBean UserRecentResourceService mockUserRecentResourceService;
+  @MockBean UserServiceAuditor mockUserServiceAuditor;
+  @MockBean WorkspaceService mockWorkspaceService;
+  @MockBean WorkspaceAuthService mockWorkspaceAuthService;
+
+  @Qualifier(NotebooksConfig.USER_DISKS_API)
+  @MockBean
+  DisksApi userDisksApi;
+
+  @MockBean ProxyApi proxyApi;
+
+  @Autowired CdrVersionDao cdrVersionDao;
+  @MockBean WorkspaceDao workspaceDao;
+  @Autowired UserDao userDao;
+  @Autowired DiskController diskController;
+  @Autowired LeonardoMapper leonardoMapper;
+
+  private DbCdrVersion cdrVersion;
+
+  @BeforeEach
+  public void setUp() {
```

Comment:
Some test preparation is unnecessary.
Try to remove `stubGetFcWorkspace`, `createFcWorkspace` and `cdrVersion` stuff, if that still work?